.circleci/config.yml

```diff
@@ -285,7 +287,7 @@ jobs:
           command: .circleci/enforce_decreasing_lint_errors.sh
       # a new cache has to be generated on every run because they are immutable
       - save_cache:
-          key: lint-cache-v4-{{ checksum "requirements-lint.txt" }}-{{ .Branch }}-{{ .BuildNum }}
+          key: lint-cache-v2-{{ .Branch }}-{{ .BuildNum }}
```

Comment:
ah, nice, because buildnum always change we don't need the requirements checksum.